test/optimize.jl

```diff
@@ -0,0 +1,90 @@
+
+using ReTest
+
+@testset "optimize" begin
+    seed = (0x38bef07cf9cc549d, 0x49e2430080b3f797)
```

Comment:
What is the reason for using this non-standard RNG here?

Reply:
If it's for reproducbility, then StableRNG.jl is your friend.

Reply:
Ah yeah, I forgot about that one. Done.